autotest/README.markdown

```diff
@@ -1,17 +1,17 @@
 # Continuous Testing
 
-This directory contains configurations for continuous testing, using
-either autotest (part of the ZenTest gem) or watchr (in the watchr
-gem).  The purpose of these tools is to automatically run the
-appropriate test when a file is changed or, if appropriate, all
-tests.  In general, they do a straightforward mapping from a given
-code file to its unit and/or integration test.
-
-It is highly recommended that you have one of these running at all
-times during development, as they provide immediate and continuous
-feedback as to your development process.  There are some general
-usability downsides as you have to track the running process, but
-those downsides are easily worth it.
+This directory contains configurations for continuous testing,
+using autotest (part of the ZenTest gem).  The purpose of this
+tool is to automatically run the appropriate test when a file is
+changed or, if appropriate, all tests.  In general, it does a
+straightforward mapping from a given code file to its unit and/or
+integration test.
+
+It is highly recommended that you have some sort of automatic test
+runner running at all times during development, as it will provide
+immediate and continuous feedback as to your development process.
+There are some general usability downsides as you have to track
+the running process, but those downsides are easily worth it.
 
 # How to use
```

Comment:
I get an error when running `autotest` from the root of the puppet repo:

```
$ autotest -b -v -n
loading autotest/puppet_rspec
Error loading Autotest style autotest/puppet_rspec (cannot load such file -- autotest/puppet_rspec). Aborting.
```
If I run from the `autotest` subdirectory, then it loads, but it doesn't run any tests when the source changes:

```
$ autotest -b -v -n
No tests matched README.markdown
No tests matched discover.rb
No tests matched images/fail.png
No tests matched images/pass.png
No tests matched images/pending.png
No tests matched puppet_rspec.rb
No tests matched rspec.rb
2018-11-01 14:43:31.694 growlnotify[62462:14602290] Failed to register with (null)
```

Reply:
I haven't actually tried anything autotest related. I had only tried the watchr stuff and found it broken. We might be better off removing the autotest directory entirely.

Reply:
> removing the autotest directory entirely

👍